graphql2/schema.graphql

```diff
@@ -435,6 +460,14 @@ type Schedule {
   isFavorite: Boolean!
 }
 
+type CalendarSubscription {
```

Comment:
Missing `notificationMinutes`, `scheduleID` and `schedule`
https://github.com/target/goalert/compare/calendar-subscription-api#diff-04e3af9ffc876a383b6f192a8ff8450dR282